### PR TITLE
tech-doc_navbar in top-left quadrant

### LINKED ISSUE #355: Tests failed in the Technical Documentation Page (15/16)
#### Browser Information

* Browser Name, Version: Chrome
* Operating System: Windows 10
* Mobile, Desktop, or Tablet:  Laptop

#### Your Code / Link to Your Pen

```
 https://codepen.io/whitebeard4708/pen/MXJRpp
 
 
```

#### Screenshot

image

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [x] Closes currently open issue: Closes #355 

#### Description

The tests require that the navbar remain positioned in the top-left quadrant of the screen. This update is true to the wording in the user story, accommodating projects with greater offsets on top and left.

### CODE REVIEW

File: src/project-tests/technical-docs-tests.js [hunk 1]
```diff
@@ -349,35 +349,36 @@ export default function createTechnicalDocsPageTests() {
...
         var navbar = document.getElementById('navbar');
-        assert.approximately(
+        // navbar should be in top-left quadrant
+        // navbar on left side
+        assert.isBelow(
           navbar.getBoundingClientRect().left,
-          0,
-          10,
+          (windowWidth / 2) - navbar.getBoundingClientRect().width,
           'Left of bounding rectangle is not correct.'
```
Comment (tchaffee) [inline src/project-tests/technical-docs-tests.js]: I know this isn't your change, but while you are here could you make this message clearer? I don't think it makes sense to Campers. What's a bouding rectangle? I think it should be something more like "The element with id="navbar" is not on the left side of the screen". What do you think?

File: src/project-tests/technical-docs-tests.js [hunk 2]
```diff
@@ -349,35 +349,36 @@ export default function createTechnicalDocsPageTests() {
...
         );
-        assert.approximately(
+        // navbar above middle
+        assert.isBelow(
           navbar.getBoundingClientRect().top,
-          0,
-          10,
+          windowHeight / 2,
           'Top of bounding rectangle is not correct.'
```
Comment (tchaffee) [inline src/project-tests/technical-docs-tests.js]: Same for this message.

File: src/project-tests/technical-docs-tests.js [hunk 3]
```diff
@@ -349,35 +349,36 @@ export default function createTechnicalDocsPageTests() {
...
         );
         window.scroll(0, 1000);
-        assert.approximately(
+        assert.isBelow(
           navbar.getBoundingClientRect().left,
-          0,
-          10,
+          (windowWidth / 2) - navbar.getBoundingClientRect().width,
           'After scroll: Left of bounding rectangle is not correct.'
```
Comment (tchaffee) [inline src/project-tests/technical-docs-tests.js]: Same for this message.

File: src/project-tests/technical-docs-tests.js [hunk 4]
```diff
@@ -349,35 +349,36 @@ export default function createTechnicalDocsPageTests() {
...
         );
-        assert.approximately(
+        assert.isBelow(
           navbar.getBoundingClientRect().top,
-          0,
-          10,
+          windowHeight / 2,
           'After scroll: Top of bounding rectangle is not correct.'
```
Comment (tchaffee) [inline src/project-tests/technical-docs-tests.js]: Same for this message.

**Comment by tbushman** [comment]
I don't have a lot of time actually. If you want to do all that you can.

**Comment by tchaffee** [comment]
@tbushman I'm away for the next week and don't have time either, so I'm just merging this for now.

**Pull request merged**